src/sbvr-api/sbvr-utils.ts

```diff
@@ -83,6 +83,7 @@ export {
 	addPureHook,
 	addSideEffectHook,
 } from './hooks';
+export { addTaskHandler } from '../tasks';
```

Comment:
I don't think sbvr-utils should be re-exporting this based on the idea that the tasks should be compatible with being a pluggable module where you could have it with full fledged functionality without being part of this repo